spec/models/profile_presenter_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'spec_helper'
+
+describe ProfilePresenter do
+  let(:user) { create(:user, email: "example@example.com") }
```

Comment:
Should we use `Faker::Internet.email` here and do `expect(subject.email).to eq(user.email)` on line 29? (I can never decide what the best thing to do is in cases like this, so I'm genuinely asking.)

Reply:
I was taught that sometimes it's best / most readable to check for something we know we put there, and can see in the spec. There are different mini-schools of thought, and so far it hasn't convinced me that one is The Best in any significant way. 🤷‍♀️